README.md

```diff
@@ -672,6 +672,237 @@ but if you need to do this manually, you can use the exposed functions for this:
 
 # Dapp-core exports
 
+Since version 2.0, dapp-core does not have a default export object.
+You have to import everything from its own separate module. Below you can find all the exports.
+
+You can either import everything from a module, or if you really want to make sure you're not importing anything
+that is not used, you can import everything from its own file.
+
+You can either go into their specific folder in the module for extra trimming, or import everything together.
+
+for example, these 2 imports are both valid:
+
+```
+import { useExtensionLogin, useGetAccountInfo } from '@elrondnetwork/dapp-core/dist/hooks';
```

Comment:
maybe we should get rid of the dist folder first?

Reply:
next in line, yes